src/ol/source/xyzsource.js

```diff
@@ -8,7 +8,19 @@ goog.require('ol.source.TileImage');
 
 /**
  * @classdesc
- * Layer source for tile data with URLs in a set XYZ format.
+ * Layer source for tile data with URLs in a set XYZ format that are
+ * defined in a URL template. By default, this follows the widely-used
+ * Google grid where `x` 0 and `y` 0 are in the top left. Grids like
+ * TMS where `x` 0 and `y` 0 are in the bottom left can be used by
+ * using the `-y` placeholder, but note that this may not work for
```

Comment:
Maybe better:

using the `{-y}` placeholder in the URL template, if no custom tile grid is configured. Otherwise, `{@link ol.source.TileImage}` can be used